### ISSUE TITLE
Configure Owlbot to ignore proto files

### ISSUE THREAD

**Issue by dansaadati** [googleapis/docuploader#109]
We should configure Owlbot to ignore proto files, and other generated files, as they do not require any post-processing.

**Comment by tbpg** [comment]
This seems to be causing an issue:

```
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
```

OwlBot is configured to format files here:
https://github.com/googleapis/docuploader/blob/993badb47be3bf548a4c1726658eadba4bafeaca/owlbot.py#L26

The `nox -s blacken` session is here:
https://github.com/googleapis/docuploader/blob/993badb47be3bf548a4c1726658eadba4bafeaca/noxfile.py#L20-L22

Looks like we need to add `--extend-exclude` to the `black` invocation to ignore the proto directory.

**Comment by tbpg** [comment]
On second look, this might be a more widespread issue and not related to the formatting? Looks like we want `protobuf==3.20.1` for now.

**Comment by tbpg** [comment]
Fixed by #110 (forgot to link when sending the PR).
